crypto/asn1/charmap.pl

```diff
@@ -7,6 +7,9 @@
 # https://www.openssl.org/source/license.html
 
 use strict;
+use FindBin;
+use lib "$FindBin::Bin/util/perl";
```

Comment:
FindBin finds modules relative to the calling script. Therefore, in this script, this use statement should be:

``` perl
use lib "$FindBin::Bin/../../util/perl";
```

Reply:
nothing that finds modules relative to a env var, such as top of tree?  sigh, okay.  thanks!

Reply:
Nope.  `FindBin` basically gives you the same as `dirname($0)` in a bash script, you gotta figure out the rest yourself.

BTW, [there's a manual :wink:](https://perldoc.perl.org/FindBin)

Reply:
Oh man, I just now realized it's a *variable.*  Thanks.